functions/packages.bash

```diff
@@ -141,473 +180,467 @@ homegear_setup() {
   fi
-  if ! cond_redirect apt-get install --yes mosquitto mosquitto-clients; then echo "FAILED"; return 1; fi
   if [ "$mqttpasswd" != "" ]; then
     if ! grep -q password_file /etc/mosquitto/passwd /etc/mosquitto/mosquitto.conf; then
       echo -e "\\npassword_file /etc/mosquitto/passwd\\nallow_anonymous false\\n" >> /etc/mosquitto/mosquitto.conf
     fi
     echo -n "" > /etc/mosquitto/passwd
-    if ! cond_redirect mosquitto_passwd -b /etc/mosquitto/passwd "$mqttuser" "$mqttpasswd"; then echo "FAILED"; return 1; fi
+    if cond_redirect mosquitto_passwd -b /etc/mosquitto/passwd "$mqttuser" "$mqttpasswd"; then echo "OK"; else echo "FAILED"; return 1; fi
   else
-    cond_redirect sed -i "/password_file/d" /etc/mosquitto/mosquitto.conf
-    cond_redirect sed -i "/allow_anonymous/d" /etc/mosquitto/mosquitto.conf
-    cond_redirect rm -f /etc/mosquitto/passwd
+    if ! cond_redirect sed -i "/password_file/d" /etc/mosquitto/mosquitto.conf; then echo "FAILED"; return 1; fi
+    if ! cond_redirect sed -i "/allow_anonymous/d" /etc/mosquitto/mosquitto.conf; then echo "FAILED"; return 1; fi
+    if cond_redirect rm -f /etc/mosquitto/passwd; then echo "OK"; else echo "FAILED"; return 1; fi
   fi
-  cond_redirect systemctl enable mosquitto.service
-  if cond_redirect systemctl restart mosquitto.service; then echo "OK"; else echo "FAILED"; return 1; fi
+
+  echo -n "$(timestamp) [openHABian] Setting up MQTT service... "
+  if ! cond_redirect systemctl enable mosquitto.service; then echo "FAILED (enable service)"; return 1; fi
+  if cond_redirect systemctl restart mosquitto.service; then echo "OK"; else echo "FAILED (restart service)"; return 1; fi
 
   if [ -n "$INTERACTIVE" ]; then
-    if [ $FAILED -eq 0 ]; then
-      whiptail --title "Operation Successful!" --msgbox "$successtext" 15 80
-    else
-      whiptail --title "Operation Failed!" --msgbox "$failtext" 10 60
-    fi
+    whiptail --title "Operation Successful!" --msgbox "$successtext" 13 80
   fi
 }
 
 find_setup() {
-  introtext="Install and setup the FIND server system to allow for indoor localization of WiFi devices. Please note that FIND will run together with an app that's available on Android ONLY. See further information at http://www.internalpositioning.com"
-  failtext="Sadly there was a problem setting up the selected option.\\nPlease report this problem in the openHAB community forum or as an openHABian GitHub issue."
-  successtext="FIND setup was successful. Settings can be configured in '/etc/default/findserver'. Be sure to restart the service after.\\nObtain the FIND app for Android through the Play Store. Check out your FIND server's dashboard at: http://${HOSTNAME}:8003\\nFor further information: http://www.internalpositioning.com"
+  if [ -z "$INTERACTIVE" ]; then
+    echo "$(timestamp) [openHABian] FIND setup must be run in interactive mode! Canceling FIND setup!"
```

Comment:
same as with nodered, not trigger from unattended installs. Or BATS?

Reply:
BTW: #632 is still open to upgrade to FIND 3

Reply:
I forgot about that, I can probably integrate that here.

Reply:
Same reason as nodered habit and future proofing.